lldb/source/Plugins/ObjectFile/ELF/ObjectFileELF.cpp

```diff
@@ -1854,6 +1854,49 @@ class VMAddressProvider {
 };
 }
 
+namespace {
```

Comment:
Nit: LLVM and LLDB prefer static functions over anonymous namespaces.

Reply:
This file already uses anonymous namespaces, but I'll change the function to `static` instead.